firebase-firestore/src/main/java/com/google/firebase/firestore/local/LocalDocumentsView.java

```diff
@@ -108,6 +115,14 @@ Document getDocument(DocumentKey key) {
     return getLocalViewOfDocuments(docs, new HashSet<>());
   }
 
+  /** Gets the local view of the next {@code count} documents based on their read time. */
+  ImmutableSortedMap<DocumentKey, Document> getDocuments(
```

Comment:
Do we still need this method? If so, we should move it back to its original spot to reduce diff.